### DISCUSSION TITLE
789

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#962] [Comments]
# 789

Java 21이 2023년 9월에 출시 된지 1년이 되어가고 있고, 여러 신규 기능이 패치 되었기도 하여서 개인 노트북에 JDK 21 설치를 하기로 했다.whichjdk.com를 보면 크게 2가지 버전의 JDK를 추천한다.Amazon CorrettoAWS의 Amazon Linux 2에서 Java 애플리케이션을 직접 실행하는 경우 최적화된 버전Adoptium Eclipse Temurin오픈 소스 소프트웨어에 대한 리소스와 전문 거버넌스 모델을 제공하는 Eclipse 재단 산하의 최상위 프로젝트Red Hat, IBM, Microsoft, Azul, iJUG 등 Java 기술에 전략적 관심을 갖고 있는 주요 기업 및 조직으로 구성이전의 AdoptOpenJDK 프로젝트는 Eclipse Adoptium으로 이전됨..

https://jojoldu.tistory.com/789

**Comment by jws1837** [comment]
자바개발환경 설치의 치트시트!

**Comment by pine14** [comment]
~/.zsh 가 ~/.zshrc 말씀하시는 걸까요?

**Reply by jojoldu2inflab** [reply]
앗 ㅋㅋㅋㅋ 맞습니다 ㅋㅋㅋㅋㅋㅋ 감사합니다 ㅋㅋㅋ
